### PR TITLE
Prints generated Dockerfile lines nicely

### PR DESCRIPTION
**Pull request by youreddy**
Resolves https://github.com/deislabs/porter/issues/111
/cc @ryanmoran

### CODE REVIEW

File: pkg/porter/build_test.go [hunk 1]
```diff
@@ -39,6 +40,36 @@ func TestPorter_buildDockerfile(t *testing.T) {
 	assert.Equal(t, wantlines, gotlines)
 }
 
+func TestPorter_buildDockerfile_output(t *testing.T) {
+	buffer := bytes.NewBuffer([]byte{})
```
Comment (carolynvs) [inline pkg/porter/build_test.go]: I would like to show you a special function that I built into TestPorter that assists with collecting output during a test.

You can remove lines 44 and 46. Then on line 70, change `buffer.String` to `p.TestConfig.TestContext.GetOutput()`. 

The TestPorter struct does the same thing that you did in the test (make a byte buffer) and then exposes it as a string with `GetOputput()`.

**Comment by carolynvs** [comment]
I'm excited to have you contributing to porter! Please let me know if you are interested in more issues, and I can help you find another. 😀

File: pkg/porter/build_test.go (hunk 1 above)
Comment (youreddy) [reply pkg/porter/build_test.go]: That makes sense to me! Just updated the branch to reflect those changes.

**Comment by youreddy** [comment]
> I'm excited to have you contributing to porter! Please let me know if you are interested in more issues, and I can help you find another. 😀

@carolynvs thank you for being so welcoming. We're excited to be contributing in this space.

**Pull request merged**